lib/game/pinball_game.dart

```diff
@@ -314,7 +315,7 @@ class _DebugInformation extends Component with HasGameRef<DebugPinballGame> {
   @override
   void render(Canvas canvas) {
     final debugText = [
-      'FPS: ${gameRef.fps().toStringAsFixed(1)}',
+      'FPS: ${gameRef.descendants().whereType<FpsComponent>().first.fps}',
```

Comment:
The the `descendants` is quite a heavy operation to be called in the render method, we should memoized the `FpsComponent` so we query for it only once.

Reply:
Updated in following commit